src/components/Header.jsx

```diff
@@ -0,0 +1,23 @@
+import { NavLink, useLocation } from 'react-router-dom';
+import Icon from '../assets/icon.png';
+import BackIcon from '../assets/icon50.png';
+import Search from './Search';
+
+const Header = () => {
+  const location = useLocation();
+  const isDetailsPage = location.pathname === '/details';
+  return (
+    <header className="header text-white d-flex justify-content-between align-items-center">
```

Comment:
- [ ] According to the design guidelines, you should follow the 
  * Typography: font face, size, and weight
  * Layout: composition and space between elements.
Please add the nav bar as seen in the original design. It's layout should be similar.
### Your design 
![image](https://github.com/Gh-Reza/weather-app/assets/45460192/0787915f-69c4-4d03-866d-fbcf00cd7ed8)
### Original
![image](https://github.com/Gh-Reza/weather-app/assets/45460192/2954e1d3-125d-4b2c-affd-40baafa11a51)